src/reducers/console.js

```diff
@@ -34,6 +34,15 @@ export default function console(stateIn, {type, payload, meta}) {
           payload.compiledProjectKey,
         ),
       );
+
+    case 'ADD_RUNTIME_ERROR':
+      return state.set(
+        payload.key,
+        new ConsoleEntry({
+          evaluatedByCompiledProjectKey: payload.compiledProjectKey,
+          error: new ConsoleError({name: "Runtime Error", message: payload.error.text})
```

Comment:
So this is a tricky situation—right now if you type code directly into the Popcode console that results in an error, the error class and message are displayed as-is from the interpreter.

However if you were to type the exact same code into the project JavaScript, the error would be “normalized”, i.e. translated into a human-readable message (when recognized).

That seems like an odd inconsistency—I think in the immediate term we probably want to always display raw errors in the console, regardless of what triggered them. There’s already [a separate issue](https://github.com/popcodeorg/popcode/issues/1260) to then start translating the errors displayed in the console across the board.

What do you think? I believe this would require adding the original error as additional information in the `ADD_RUNTIME_ERROR` payload, but that seems reasonable enough.

Reply:
Can you give me an example? I'm not following what you want to happen. I do see they are different though. Top error is from the javascript section, the bottom is from the console. 
![screenshot 2018-08-29 11 27 44](https://user-images.githubusercontent.com/11447584/44798621-f71efe80-ab7f-11e8-8cee-32fe09584e7c.png)